Samples.xcodeproj/project.pbxproj

```diff
@@ -514,12 +516,12 @@
 		00E5400B27F3CCA100CF66D5 /* Shared */ = {
 			isa = PBXGroup;
 			children = (
-				0074ABAF281742420037244A /* Supporting Files */,
```

Comment:
Please revert this change. The upper level folders don't need to be sorted, only the samples folders need to.